vspace/src/main/java/edu/asu/diging/vspace/web/staff/StaffSearchController.java

```diff
@@ -0,0 +1,117 @@
+package edu.asu.diging.vspace.web.staff;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import edu.asu.diging.vspace.core.model.impl.Module;
+import edu.asu.diging.vspace.core.model.impl.Slide;
+import edu.asu.diging.vspace.core.model.impl.Space;
+import edu.asu.diging.vspace.core.services.IStaffSearchManager;
+
+@Controller
+public class StaffSearchController {
+
+    @Autowired
+    private IStaffSearchManager staffSearchManager;
+
+    @RequestMapping(value = "/staff/search")
+    public String searchInVspace(
+            @RequestParam(value = "spacePagenum", required = false, defaultValue = "1") String spacePagenum,
+            @RequestParam(value = "modulePagenum", required = false, defaultValue = "1") String modulePagenum,
+            @RequestParam(value = "slidePagenum", required = false, defaultValue = "1") String slidePagenum,
+            @RequestParam(value = "slideTextPagenum", required = false, defaultValue = "1") String slideTextPagenum,
+            Model model, @RequestParam(name = "searchText") String searchTerm) {
+
+        paginationForSpace(spacePagenum, model, searchTerm);
+
+        paginationForModule(modulePagenum, model, searchTerm);
+
+        paginationForSlide(slidePagenum, model, searchTerm);
+
+        paginationForSlideText(slideTextPagenum, model, searchTerm);
+
+        model.addAttribute("searchWord", searchTerm);
+        
+        return "/staff/search/staffSearch";
+    }
+
+    /**
+     * This method is used to search the searched string specified in the input
```

Comment:
That sounds very odd and sounds like you are searching in the string not for it.. Either make it "search string" or just say string.